.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Run Jest tests
+on:
+  push:
+    branches:
+      - feat/manager-shell
```

Comment:
**NB**: Code to avoid pollution on other pull requests. This restriction will be removed after approvals